doc/docs/overview.md

```diff
@@ -16,66 +20,106 @@ Opens a dialog with localized links to partner websites to purchase the supporte
 
-### Serial Port Trace Capture
+#### Modem trace serial port
 
 The serial port used to send the modem trace. It is the last port in the dropdown list if your application firmware uses default settings.
 
-### Reset Device
+#### Reset device on start
+
+If toggled on, this option will cause the device to reset before starting a new trace.
+
+#### Refresh dashboard on start
 
-Resets the device before starting a new trace.
+If toggled on, this option will refresh the dashboard status automatically when you [start tracing](#start).
 
-### Open in Wireshark
+#### Open in Wireshark
 
-Opens and displays trace data live in Wireshark when the trace is started.
+If toggled on, this option will automatically detect your Wireshark installation. If you have Wireshark installed, it will open and display trace data live in Wireshark when the trace is started.
+See [Using Wireshark](./wireshark.md) for more information.
 
-### Program Device
+#### Save trace file to disk
 
-Select and program precompiled sample applications and modem firmware to your device. The samples enable the trace and [AT Host](https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/index.html) prerequisites for **Cellular Monitor**. Modem firmware supporting trace is available with all samples, and you can choose to program the modem or application firmware, or both. The modem firmware needs only to be programmed once.
+If toggled on, starting tracing will create a raw temporary trace file for storing a copy of the trace for future use. After you start tracing, the information about the file and its size appears under the toggle.
 
-### Terminal Serial Port
+![Tracing file size information](./screenshots/cel_mon_overview.png "Tracing file size information")
 
-The serial port used to send AT commands to your device if Modem Shell or [AT Host](https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/index.html) is enabled. This is the port used by Serial Terminal. It is the first port in the dropdown list if your application firmware uses default settings.
+Clicking the file name opens its location.
 
-### Dashboard Tab and Packet Event Viewer
+### Advanced options
 
-See [Viewing a Modem trace in Cellular Monitor](./viewing.md).
+This section lists advanced tracing options.
 
-### Certificate Manager Tab
+#### Program device
+
+Select and program precompiled sample applications and modem firmware to your device. The samples enable the trace and [AT Host](https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/index.html) prerequisites for Cellular Monitor. Modem firmware supporting trace is available with all samples, and you can choose to program the modem or application firmware, or both. The modem firmware needs only to be programmed once.
```

Comment:
Is it also worth mentioning that if you choose to program the modem firmware, then you need to program the application firmware as well?

More specifically, programming the modem firmware seems to delete the application firmware.

Reply:
I was thinking if the customer has a custom firmware, and just wants to update the modem firmware using the app, it's not easy for the app to handle that case. But I assume this is very corner-case.

I wrote "seems to delete", because I'm not entirely sure if that should be the behavior, but it's what we have experienced in the past.

Example:
- user develops their own app
- programs the application to device
- decides to update modem firmware, and programs the new mfw to app
--> now the application is wiped from device, so user must again go and program the application to device.